src/xarray_regrid/regrid.py

```diff
@@ -55,84 +57,69 @@ def cubic(
+        return methods.interp_regrid(self._obj, ds_target_grid, "cubic")
 
-    def nearest(
+    def conservative(
         self,
         ds_target_grid: xr.Dataset,
+        latitude_coord: str | None,
         time_dim: str = "time",
-    ) -> xr.Dataset:
-        """Return a dataset regridded by taking the values of the nearest target coords.
+    ) -> xr.DataArray | xr.Dataset:
+        """Regrid to the coords of the target dataset with a conservative scheme.
 
         Args:
             ds_target_grid: Dataset containing the target coordinates.
-            time_dim: The name of the time dimension/coordinate
+            latitude_coord: Name of the latitude coord, to be used for applying the
+                spherical correction.
+            time_dim: The name of the time dimension/coordinate.
 
         Returns:
-            Dataset regridded to the target dataset coordinates.
+            Data regridded to the target dataset coordinates.
         """
-        ds_target_grid = validate_input(self._obj, ds_target_grid, time_dim)
-        return methods.interp_regrid(self._obj, ds_target_grid, "nearest")
 
-    def cubic(
-        self,
-        ds_target_grid: xr.Dataset,
-        time_dim: str = "time",
-    ) -> xr.Dataset:
         ds_target_grid = validate_input(self._obj, ds_target_grid, time_dim)
-        return methods.interp_regrid(self._obj, ds_target_grid, "cubic")
+        return methods.conservative_regrid(self._obj, ds_target_grid, latitude_coord)
 
-    def conservative(
+    def most_common(
         self,
         ds_target_grid: xr.Dataset,
-        latitude_coord: str | None,
         time_dim: str = "time",
-    ) -> xr.Dataset:
+        max_mem: int = int(1e9),
+    ) -> xr.DataArray | xr.Dataset:
+        """Regrid by taking the most common value within the new grid cells.
+
+        To be used for regridding data to a much coarser resolution, not for regridding
+        when the source and target grids are of a similar resolution.
+
+        Note that in the case of two unqiue values with the same count, the behaviour
+        is not deterministic, and the resulting "most common" one will randomly be
+        either of the two.
+
+        Args:
+            ds_target_grid: Target grid dataset
+            time_dim: Name of the time dimension. Defaults to "time".
+            max_mem: (Approximate) maximum memory in bytes that the regridding routine
+                can use. Note that this is not the total memory consumption and does not
+                include the size of the final dataset. Defaults to 1e9 (1 GB).
+
+        Returns:
+            Regridded data.
+        """
         ds_target_grid = validate_input(self._obj, ds_target_grid, time_dim)
-        return methods.conservative_regrid(self._obj, ds_target_grid, latitude_coord)
+        return most_common.most_common_wrapper(
+            self._obj, ds_target_grid, time_dim, max_mem
+        )
```

Comment:
I think the user should receive a warning when the given data set can not cover the target grid. So it is better to have an extra check here. Currently, if that is the case, the regridder still works, but user will get a field with only 0 outside of the area covered by the original data.

We can also create an issue and leave it for later.

Reply:
I created an issue